index.html

```diff
@@ -4380,6 +4381,125 @@ <h3>Parsing content</h3>
+      Scanning a tag with NFC requires the user to place the scanning device
+      (e.g. phone) in close proximity to the NFC tag.
+    </p>
+    <p>
+      Scanning a tag when a Web NFC scan is not active, triggers the host OS
+      handling. Thus launching URLs and apps from scanning a NFC tag is not
+      handled and supported by Web NFC itself.
+    </p>
+    <p>
+      Furthermore, Web NFC scanning needs to be activated from a user interaction,
+      and scanning is paused when the web site is not in focus or the device
+      screen turns off (i.e. is not unlocked). This is put in place so that
+      accidental scans do not happen.
+    </p>
+    <p>
+      Web NFC further recommends that the implementation makes it very clear
+      UX-wise to the user that data will be scanned when placing the scanning
+      device in close proximity to a NFC tag - basically mimicking the UX flow
+      of scanning a QR code.
+    </p>
+    <p>
+      There are many options for doing so, like playing back a sound or showing
+      some persistent UI while the scanning can happen, for instance a modal
+      dialog with the ability to cancel at any point.
+    </p>
+    <img src="images/scan.jpg">
+    <p>
+      An implementation could also show the data that is about to be uploaded,
+      postpone sharing the read data until the user OK'd it and even show some
+      UI allowing the user to select which records to share.
+    </p>
+    <img src="images/approve.jpg">
+    <section> <h4>Reading and writing during a scan</h4>
+      <p>
+       When the user scans a tag, at that point the web application has access
+       to read the data on the tag, and in case it is not read-only, also to
+       write data to the tag. Consumer stickers for private usage (e.g. in the
+       maker community) are often unlocked (read + write), whereas commercial
+       deployment of NFC are read-only.
+      </p>
+      <p class="note">
+        An older protocol SNEP (Simple NDEF Exchange Protocol) allowed active
+        devices (e.g. a phone) to receive NDEF data from another active device,
+        but it is unsupported by Web NFC and it is currently being deprecated
+        on supported native platforms.
+      </p>
+      <p class="note">
+        A newer protocol TNEP (Tag NDEF Exchange Protocol) allows bidirectional
+        communication between a scanner device (e.g. phone) and actively powered
+        device like an IOT device. It is currently unsupported by Web NFC, and
+        furthermore it has restrictions on what input to accept and the IOT device
+        must ensure that the accepted records are valid.
+      </p>
+      <p>
+        If the tag contains privacy sensitive data, such data will be shared with
+        the site. Potentially, not immediately if the UX requires the user to
+        confirm the data exchange before doing so.
+      </p>
+      <p>
+        In some cases it might be obvious that the tag/device contains privacy
```

Comment:
How "obvious" is it?

Reply:
for conference badges, business cards etc it is quite obvious

Reply:
rewrote

Reply:
the NFC tag embedded into a device